benchmark/solver/batch_solver.hpp

```diff
@@ -295,8 +295,8 @@ std::unique_ptr<gko::BatchLinOpFactory> generate_solver(
     if (description == "richardson") {
         using Solver = gko::solver::BatchRichardson<etype>;
         return Solver::build()
-            .with_max_iterations(static_cast<int>(FLAGS_max_iters))
-            .with_residual_tol(
+            .with_default_max_iterations(static_cast<int>(FLAGS_max_iters))
```

Comment:
I think this is a nice idea. But `default` may be a bit confusing. How about `prescribed` or `prescr` for short ?

Reply:
I guess `default` is also okay. So you can leave it as it is.